go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/BurntSushi/toml v1.1.0
 	github.com/celestiaorg/celestia-app v0.5.2
 	github.com/celestiaorg/go-libp2p-messenger v0.1.0
-	github.com/celestiaorg/nmt v0.8.0
+	github.com/celestiaorg/nmt v0.9.0
```

Comment:
there is a v0.10.0 now. Also, node should use the same version which the core/app combo uses that node depends on imo.

Reply:
yes

Reply:
We can wait for dependabot to realize there is a new version, or we can update ourselves. I would opt for dependabot as this does not block anything.